amdgpu-monitor

```diff
@@ -177,7 +178,8 @@ def main():
         else:
             gpu_list.print()
         sys.exit(0)
-
```

Comment:
For example, those validation logic could be in a separate module/class/function. GUI and Text could be split into 2 module/class/function. Then there'll be just several lines of code in your main() function.

Reply:
Makes sense.  I will consider this when I split amdgpu-ls from amdgpu-monitor.

Are you able to see and comment on my pull request ask for help?

Reply:
I made major changes to move all gtk code to class functions, resulting in simplified main functions for both amdgpu-monitor and amdgpu-pac.